src/aind_slims_api/__init__.py

```diff
@@ -7,3 +7,28 @@
 config = AindSlimsApiSettings()
 
 from aind_slims_api.core import SlimsClient  # noqa
+
+
+def testmod(**testmod_kwargs):
```

Comment:
It will probably be better to add code to facilitate testing in the testing module

Reply:
resolved in [d5e6d27](https://github.com/AllenNeuralDynamics/aind-slims-api/pull/20/commits/d5e6d27f39eda480803b55daf2f30d461008b877)